### DISCUSSION TITLE
macOS special characters

### DISCUSSION THREAD

**Discussion by danielfalbo** [qutebrowser/qutebrowser#5834] [Q&A] [answered]
On macOS, when you hold down a key, it will display a popup allowing you to pick a special character if there are any assigned to that particular key. 
![](https://www.howtogeek.com/wp-content/uploads/2016/08/img_57b1f6e2a50d0.png)
However, in qutebrowser this seems not to work.
![in](https://user-images.githubusercontent.com/39460524/97166458-94d4df00-1785-11eb-8d6a-a362a47ada51.gif)
Does anyone have a workaround or something?

**Comment by The-Compiler** [comment]
Do you have any idea how this popup is called (i.e. what to search for)? Do you happen to have another Qt application around to test there? (perhaps Virtualbox or Telegram Desktop?)

**Reply by danielfalbo** [reply]
Looks like they call it the ["accent menu"](https://support.apple.com/guide/mac-help/enter-characters-with-accent-marks-on-mac-mh27474/mac).
Seems to work in Telegram: 
![in](https://user-images.githubusercontent.com/39460524/97203875-b057de00-17b5-11eb-8bc0-655f20faf221.gif).

**Comment by danielfalbo** [comment, marked as answer]
Oh, [another similar issue](https://github.com/equalsraf/neovim-qt/issues/617), looks like a Qt Bug: https://bugreports.qt.io/browse/QTBUG-71394 .

**Reply by The-Compiler** [reply]
Looks like that is the culprit indeed. Telegram ships with a patched Qt as far as I remember, so maybe they applied the patch which was mentioned in the issue or something.
